core/cmd/local_client_test.go

```diff
@@ -439,6 +431,22 @@ func TestClient_RebroadcastTransactions_Txm(t *testing.T) {
 		Logger:                 lggr,
 	}
 
+	beginningNonce := uint(7)
+	endingNonce := uint(10)
+	gasPrice := big.NewInt(100000000000)
+	gasLimit := uint64(3000000)
+	set := flag.NewFlagSet("test", 0)
+	cltest.FlagSetApplyFromAction(client.RebroadcastTransactions, set, "")
+
+	require.NoError(t, set.Set("beginningNonce", strconv.FormatUint(uint64(beginningNonce), 10)))
+	require.NoError(t, set.Set("endingNonce", strconv.FormatUint(uint64(endingNonce), 10)))
```

Comment:
pre-existing and out-of-scope, but I wonder if all this explicit typing and converting could be simplified with untyped values :thinking: 
```suggestion
	beginningNonce := 7
	endingNonce := 10
	gasPrice := big.NewInt(100000000000)
	gasLimit := uint64(3000000)
	set := flag.NewFlagSet("test", 0)
	cltest.FlagSetApplyFromAction(client.RebroadcastTransactions, set, "")

	require.NoError(t, set.Set("beginningNonce", strconv.FormatUint(beginningNonce, 10)))
	require.NoError(t, set.Set("endingNonce", strconv.FormatUint(endingNonce, 10)))
```
Trade-off may be adding an explicit type for `i` below, e.g. `for var i uint = beginningNonce`. Although, using the type returned by `tx.Nonce()` instead of `uint` could save a conversion there too.

Reply:
Fixed in [44845a3](https://github.com/smartcontractkit/chainlink/pull/8173/commits/44845a3d1de9a6ac3def557c0763f53451e4f980)